palisade-service/README.md

```diff
@@ -23,16 +23,19 @@ limitations under the License.
 The Palisade Service is the entry point into the rest of the Palisade Services.
 It accepts a request from the client, containing the userId of the user requesting access, the resource the user wants access to, and the context as to why they require access. 
 The Palisade Service then creates a token, currently via the UUIDPalisadeService, but this can be extended in the future for specific token requirements. 
-The token is attached to the message in the form of a `PalisadeSystemResponse` to be sent to the User Service on the `user` kafka topic. 
+The token is attached to the message in the form of a `PalisadeSystemResponse` to be sent to the User Service on the `user` Kafka topic. 
 It is also sent back to the client directly as part of a `PalisadeClientResponse` so that the client can connect to the Filtered Resource Service to retrieve the processed request.
 As the request progresses through the other Palisade services, it is refined and enriched, the Palisade Service is the entry point for client requests.
+The service also creates a start and end marker message for each request that will help the other services to determine where each request starts and ends.
 
 ## High-Level Architecture
 <!--- 
 See palisade-service/doc/palisade-service.drawio for the source of this diagram
 --->
 ![Palisade Service diagram](doc/palisade-service.png)
-The routing of requests is shown in the diagram above. The yellow boxes indicate the client, and the purple are kafka topics.
+
+The routing of requests is shown in the diagram above. 
+The green box indicate the client request, the red box indicates if an error has occurred, and the purple boxes are Kafka topics.
```

Comment:
red should be creating errors, blue for persistence icons

Reply:
Updated